configure.ac

```diff
@@ -748,6 +748,21 @@ AC_ARG_ENABLE(
   [AS_HELP_STRING([--disable-doxygen-version],
   [Substitute the Doxygen version with latest, for the website])])
 
+# systemd support
+AC_ARG_ENABLE(
+  [systemd],
+  [AS_HELP_STRING([--enable-systemd], [Enable systemd support])])
+
+have_libsystemd="no"
+AS_IF([test "x$enable_systemd" = "xyes"],
+  [PKG_CHECK_MODULES([libsystemd], [libsystemd >= 38], [have_libsystemd="yes"])
+   AC_CHECK_HEADER([systemd/sd-daemon.h], [], [have_libsystemd="no"])])
+
+AS_IF([test "x$have_libsystemd" = xyes],
+      [AC_DEFINE([HAVE_LIBSYSTEMD], [1], [define if systemd support wanted])])
```

Comment:
Nit: I'd probably say this is if systemd support present (or present and wanted).